tests/test_install_requires.py

```
@@ -38,47 +38,48 @@
 			f.write(content)
 
-	def test_simple_version_constraint(self):
+	def test_simple_version_constraint(self) -> None:
 		"""Test parsing a simple version constraint."""
-		self.write_requirements("""package>=1.0\n""")
+		self.write_requirements("package>=1.0\n")
 		install_requires = parse_requirements_for_install_requires(
 			readFile(self.requirements_file)
 		)
-		self.assertEqual(install_requires, ["""package>=1.0"""])
+		self.assertEqual(install_requires, ["package>=1.0"])
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)